code/modules/clothing/spacesuits/miscellaneous.dm

```diff
@@ -396,6 +396,13 @@ Contains:
 	if (user.head == src)
 		user.faction -= "carp"
 
+/obj/item/clothing/suit/space/hardsuit/carp/old
+	name = "battered carp space suit"
+	desc = "It's covered in bite marks and scratches, yet seems to be still perfectly functional."
+	slowdown = 1
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0, "stamina" = 0)
```

Comment:
Give it normal armor values instead of 0

Reply:
By that I assume you mean buffing the suit's armor or give it the same armor as the original carp suit?

Reply:
No, I mean normal as in like the other space suit curator has